td-agent/Rakefile

```diff
@@ -244,6 +244,15 @@ class BuildTask
     @download_task = download_task
   end
 
+  def make_wix_version_numer(version)
+    if version.split("~")[1]
+      version_base = version.split("~")[0].chars.delete_if{|e| e == "."}.join("").to_i.pred.to_s.chars.join(".")
```

Comment:
It seems that `delete(".")` is enough instead of `delete_if{...}`.

Reply:
Oh! Good suggestion. :+1: